Specs countries (#293)
PrestaShop/prestashop-specs: content/1.7/back-office/international/locations/countries/add-edit-country.md

---
title: Add edit country
weight: 1
---

# Add edit country

When adding or editing a country, the following fields / drop-down / toggles are displayed:

>  - **Country** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The field name is required at least in English (English).". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.

Reviewer: ```suggestion
 - **Country** - This field is mandatory. When trying to save without filling this field, the following error message is displayed: "The field name is required at least in _default language_.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with the default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
```

The following tooltip is displayed: "Country name - Invalid characters: <>;=#{}".

>  - **ISO code** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).".

Reviewer: ```suggestion
 - **ISO code** - This field is mandatory. When trying to save without filling this field, the following error message is displayed: "The iso_code field is required.". If the field is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).".
```

Reviewer: What happens if you try to create a country with an ISO code already existing?

>  - **Call prefix** - This is a mandatory field. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)".

Reviewer: ```suggestion
 - **Call prefix** - This field is mandatory. The following tooltip is displayed: "International call prefix, (e.g. 1 for United States)".
```

Reviewer: What happens if you try to save without filling this field ?

>  - **Default currency** - This is a drop-down field which allows to seect a currency from the ones that are enabled on the shop.

Reviewer: ```suggestion
 - **Default currency** - This is drop-down allows to select a currency from the ones that are enabled.
```

Reviewer: What is the default value for this drop-down ?

>  - **Zone** - This is a drop-down field with all the zones that can be selected. The following tooltip is displayed: "Geographical region.".

Reviewer: ```suggestion
 - **Zone** - This drop-down allows selecting a zone. All the zones are listed (both enabled and disabled ones). The following tooltip is displayed: "Geographical region.".
```

>  - **Does it need Zip/Postal code?** - Toggle switch for either "yes" or "no". By default is set to "yes".

Reviewer: What happens on the front office if this option is on Yes or No ?

>  - **Zip/Postal code format** - This is a mandatory field.

Reviewer: ```suggestion
 - **Zip/Postal code format** - This field is mandatory.
```

> The following helptext below is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank."

Reviewer: ```suggestion
The following help text is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank."
```

>  - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."

Reviewer: ```suggestion
 - **Address format** - Firstly, a box is displayed with the following default values: firstname lastname, company, vat_number, address1, address2, postcode city, Country:name, phone. On the right, a help text is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
```

Reviewer: What happens if I save and if a parameter already existing has been added many times in the address format ?

Author: @marionf It adds for me without any issues, but not sure if it's a bug or not

> Under the helptext there are 5 fields that can be pressed, and when pressed on any of the selections will be added to the address format input box:

Reviewer: ```suggestion
Under the help text, the 5 following tabs are displayed and can be selected. After selecting a tab, different parameters are displayed and can be added to the address format. 
```

>      - **Customer** -  in this section it's possible to press the "+" sign to add to the format the following inputs: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 

Reviewer: ```suggestion
     - **Customer** -  the following fields can be added to the address format: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 
```

>      - **Warehouse** - in this section it's possible to press the "+" sign to add to the format the following inputs: reference, name, management_type, htmlFields

Reviewer: ```suggestion
     - **Warehouse** - the following parameters can be added to the address format: reference, name, management_type, htmlFields
```

>      - **Country** - in this section it's possible to press the "+" sign to add to the format the following inputs: iso_code, name, zip_code_format, htmlFields

Reviewer: ```suggestion
     - **Country** - the following parameters can be added to the address format: iso_code, name, zip_code_format, htmlFields
```

>      - **State** - in this section it's possible to press the "+" sign to add to the format the following inputs: iso_code, name, htmlFields 

Reviewer: ```suggestion
     - **State** - ithe following parameters can be added to the address format: iso_code, name, htmlFields 
```

>      - **Address** - in this section it's possible to press the "+" sign to add to the format the following inputs: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.

Reviewer: ```suggestion
     - **Address** - ithe following parameters can be added to the address format: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
```

     
 Below there are 4 buttons with different actions:

>  

Reviewer: ```suggestion
```

>      - **Use the last registered format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer: ```suggestion
     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
```

When pressed "ok" will input the address format box with the last registered formats.

>      - **Use the default format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer: ```suggestion
     - **Use the default format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
```

When pressed "ok" will input the address format box with the default address formats.

>      - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer: I don't see any modal on my side when I click on this button

Author: @marionf 
![image](https://user-images.githubusercontent.com/75423142/152989498-be06f809-b98c-42bb-98df-ba37b4000f03.png)

> When pressed "ok" will input the address format box with the current modified formats.

Reviewer: ```suggestion
```

>      - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer: ```suggestion
     - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". When "ok" is pressed, all the fields are removed from the address format.
```

> When pressed "ok" will remove all the formats.

Reviewer: ```suggestion
```

>  - **Active** - Toggle switch that can be turned to "Yes" or "No". By default is set to "yes".

Reviewer: What is the tooltip ?

>  - **Active** - Toggle switch that can be turned to "Yes" or "No". By default is set to "yes".

Reviewer: ```suggestion
 - **Active** - This toggle can be turned to "Yes" or "No" to enable or not the country. By default, it is set to "yes".
```

>  - **Contains states** - Toggle switch that can be turned to "Yes" or "No". By default is set to "no".

Reviewer: ```suggestion
 - **Contains states** - This toggle indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default is set to "no". If set on "yes", this adds a new field to the address format.
```

>  - **Do you need a tax identification number?** - Toggle switch that can be turned to "Yes" or "No". By default is set to "no".

Reviewer: ```suggestion
 - **Do you need a tax identification number?** - This toggle can be turned to "Yes" or "No". By default it is set to "no".
```

>  - **Display tax label (e.g. "Tax incl.")** - Toggle switch that can be turned to "Yes" or "No". By default is set to "yes".

Reviewer: ```suggestion
 - **Display tax label (e.g. "Tax incl.")** - This toggle allows choosing whether the tax status (included or excluded) should be displayed next to the prices or not. It can be turned to "Yes" or "No". By default, it is set to "yes".
```

---
title: Add edit country
weight: 1
---

# Add edit country

When adding or editing a country, the following fields / drop-down / toggles are displayed:

 - **Country** - This field is mandatory. When trying to save without filling this field, the following error message is displayed: "The field name is required at least in _default language_.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with the default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
The following tooltip is displayed: "Country name - Invalid characters: <>;=#{}".

>  - **ISO code** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).". If you try to add a country with an already existing ISO, the following error will show: "This ISO code already exists.You cannot create two countries with the same ISO code."

Reviewer: ```suggestion
 - **ISO code** - This field is mandatory. When trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).". If you try to add a country with an already existing ISO, the following error will show: "This ISO code already exists.You cannot create two countries with the same ISO code."
```
```suggestion
 - **ISO code** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).". If you try to add a country with an already existing ISO, the following error will show: "This ISO code already exists.You cannot create two countries with the same ISO code."
```

>  - **Call prefix** - This is a mandatory field. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to proceed without enetering this field, the following error message is displayed: "The Call prefix field is required.". 

Reviewer: ```suggestion
 - **Call prefix** - This field is mandatory. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to save without filling this field, the following error message is displayed: "The Call prefix field is required.". 
```
```suggestion
 - **Call prefix** - This is a mandatory field. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to proceed without enetering this field, the following error message is displayed: "The Call prefix field is required.". 
```

     
 Below there are 4 buttons with different actions:
 
     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
When pressed "ok" will input the address format box with the last registered formats.
     - **Use the default format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
When pressed "ok" will input the address format box with the default address formats.
     - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
     - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". When "ok" is pressed, all the fields are removed from the address format.
     

>  - **Active** - This toggle can be turned to "Yes" or "No" to enable or not the country. By default, it is set to "yes". A disabled country will not be suggested as an option when a visitor wants to register and create a new account. The tooltip displays: "Display this country to your customers (the selected country will always be displayed in the Back Office).

Reviewer: ```suggestion
 - **Active** - This toggle can be turned to "Yes" or "No" to enable or not the country. By default, it is set to "yes". A disabled country will not be suggested as an option when a visitor wants to create or edit an address. The tooltip displays: "Display this country to your customers (the selected country will always be displayed in the Back Office).
```

>  - **Contains states** - This toggle indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default is set to "no". If set on "yes", this adds a new field to the address format.

Reviewer: ```suggestion
 - **Contains states** - This toggle indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default, it is set to "no". If set on "yes", a new field allowing to select the state is displayed in the address form.
```

 - **Country** - This field is mandatory. When trying to save without filling this field, the following error message is displayed: "The field name is required at least in _default language_.". This field can be translated: When several languages are installed, a drop-down next to the field is displayed with the default language selected. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
The following tooltip is displayed: "Country name - Invalid characters: <>;=#{}".

 - **ISO code** - This is a mandatory field, when trying to proceed without enetering this field, the following error message is displayed: "The iso_code field is required.". If the field input is invalid, the following error message is displayed: "The iso_code field is invalid." The following tooltip is displayed: "Two -- or three -- letter ISO code (e.g. "us" for United States).". If you try to add a country with an already existing ISO, the following error will show: "This ISO code already exists.You cannot create two countries with the same ISO code."

>  - **Call prefix** - This field is mandatory. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to save without filling this field, the following error message is displayed: "The Call prefix field is required.". 

Reviewer 2: ```suggestion
 - **Call prefix** - This field is mandatory. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to save without filling this field, the following error message is displayed: "The Call prefix field is required.". If the field input is invalid, the following error message is displayed: "The call_prefix field is invalid.".
```

 
 - **Default currency** - This drop-down allows to select a currency from the ones that are enabled on the shop. By default, the drop-down is set to: "Default store currency".
 
 - **Zone** - This drop-down allows selecting a zone. All the zones are listed (both enabled and disabled ones). The following tooltip is displayed: "Geographical region.".
 

>  - **Does it need Zip/Postal code?** - This toggle indicates whether a user living in this country must give a zip code or not when signing up to the shop. The toggle can be put on "yes" or "no". By default, it is set to "yes". 

Reviewer 2: ```suggestion
 - **Does it need Zip/Postal code?** - This toggle indicates whether a user living in this country must give a zip code or not when signing up to the shop. The toggle can be put on "yes" or "no". By default, it is set to "No". 
```

 - **Zip/Postal code format** - This field is mandatory.

> The following help text is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank."

Reviewer 2: ```suggestion
The following help text is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank." If the field input is invalid, the following error message is displayed: "The zip_code_format field is invalid.".
```

Reviewer 2: We can save without filling this field and without any error message. Is it mandatory?

>  - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."

Reviewer 2: Could you please check all error messages for this field ? Could you please specify all cases?
https://watch.screencastify.com/v/24Cf0RdS2ouR41YjBDcr
https://watch.screencastify.com/v/K38U3qXNGvm4Xe4P6S3z
https://watch.screencastify.com/v/NIxGex4QzLibkM3NefKC

     - **Address** - ithe following parameters can be added to the address format: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
When a parameter is already existing, when trying to save it, the error shows: "This key has already been used.: phone"
     
 Below there are 4 buttons with different actions:
 
     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
When pressed "ok" will input the address format box with the last registered formats.
     - **Use the default format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
When pressed "ok" will input the address format box with the default address formats.
     - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

>      - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". When "ok" is pressed, all the fields are removed from the address format.

Reviewer 2: When trying to save after clear format the following error message is displayed :
"There are 6 errors.
1. The firstname field (in tab Address) is required.
2. The lastname field (in tab Address) is required.
3. The address1 field (in tab Address) is required.
4. The city field (in tab Address) is required.
5. The Country:name field (in tab Country) is required.
6. Address format invalid".

---

> title: Add edit country

Reviewer 3: ```suggestion
title: Add / edit  a country
```

weight: 1
---

> # Add edit country

Reviewer 3: ```suggestion
# Add / edit  a country
```

 - **Call prefix** - This field is mandatory. The following tooltip is displayed: "Internation call prefix, (e.g. 1 for United States)". When trying to save without filling this field, the following error message is displayed: "The Call prefix field is required.". If the field input is invalid, the following error message is displayed: "The call_prefix field is invalid.".
 
 - **Default currency** - This drop-down allows to select a currency from the ones that are enabled on the shop. By default, the drop-down is set to: "Default store currency".
 
 - **Zone** - This drop-down allows selecting a zone. All the zones are listed (both enabled and disabled ones). The following tooltip is displayed: "Geographical region.".
 

>  - **Does it need Zip/Postal code?** - This toggle indicates whether a user living in this country must give a zip code or not when signing up to the shop. The toggle can be put on "yes" or "no". By default, it is set to "No". 

Reviewer 3: ```suggestion
 - **Does it need Zip/Postal code?** - This toggle button indicates whether a user living in this country must give a zip code or not when signing up to the shop. The toggle button can be put on "Yes" or "No". By default, it is set to "No". 
```

 - **Zip/Postal code format** - This field is mandatory.
The following help text is displayed: "Indicate the format of the postal code: use L for a letter, N for a number, and C for the country's ISO 3166-1 alpha-2 code. For example, NNNNN for the United States, France, Poland and many other; LNNNNLLL for Argentina, etc. If you do not want PrestaShop to verify the postal code for this country, leave it blank." If the field input is invalid, the following error message is displayed: "The zip_code_format field is invalid.".

>  - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."

Reviewer 3: ```suggestion
 - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, vat_number, address1, address2, postcode city, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
```

 - "The lastname field (in tab Address) is required."
 - "The address1 field (in tab Address) is required."
 - "The city field (in tab Address) is required."
 - "The Country:name field (in tab Country) is required."
 - "Address format invalid".

     - **Customer** -  the following parameters can be added to the address format: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 
      - **Warehouse** - the following parameters can be added to the address format: reference, name, management_type, htmlFields
     - **Country** - the following parameters can be added to the address format: iso_code, name, zip_code_format, htmlFields
     - **State** - the following parameters can be added to the address format: iso_code, name, htmlFields 
     - **Address** - ithe following parameters can be added to the address format: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
When a parameter is already existing, when trying to save it, the error shows: "This key has already been used.: phone"
     
 Below there are 4 buttons with different actions:
 

>      - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer 3: ```suggestion
     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". A tooltip is displayed on button hovered: "This will restore your last registered address format"
```

When pressed "ok" will input the address format box with the last registered formats.

>      - **Use the default format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer 3: ```suggestion
     - **Use the default format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". A tooltip is displayed on button hovered: "This will restore the default address format for this country"
```

When pressed "ok" will input the address format box with the default address formats.

>      - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".

Reviewer 3: ```suggestion
     - **Use my current modified format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". A tooltip is displayed on button hovered: "This will restore your current address format"
```

>      - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". When "ok" is pressed, all the fields are removed from the address format.

Reviewer 3: ```suggestion
     - **Clear format** - When pressed, prompts a message asking: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". When "ok" is pressed, all the fields are removed from the address format. A tooltip is displayed on button hovered: "This will delete the current address format"
```

>  - **Active** - This toggle can be turned to "Yes" or "No" to enable or not the country. By default, it is set to "yes". A disabled country will not be suggested as an option when a visitor wants to create or edit an address. The tooltip displays: "Display this country to your customers (the selected country will always be displayed in the Back Office).

Reviewer 3: ```suggestion
 - **Active** - This toggle button can be turned to "Yes" or "No" to enable or not the country. By default, it is set to "Yes". A disabled country will not be suggested as an option when a visitor wants to create or edit an address. A tooltip is displayed on button name: "Display this country to your customers (the selected country will always be displayed in the Back Office).
```

>  - **Contains states** - This toggle indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default, it is set to "no". If set on "yes", a new field allowing to select the state is displayed in the address form.

Reviewer 3: ```suggestion
 - **Contains states** - This toggle button indicates whether the country has "states" or not. It can be turned to "Yes" or "No". By default, it is set to "No". If it's setted on "Yes", a new field allowing to select the state is displayed in the address form.
```

>  - **Do you need a tax identification number?** - This toggle can be turned to "Yes" or "No". By default, it is set to "no".

Reviewer 3: ```suggestion
 - **Do you need a tax identification number?** - This toggle button can be turned to "Yes" or "No". By default, it is set to "No".
```

>  - **Display tax label (e.g. "Tax incl.")** - This toggle allows choosing whether the tax status (included or excluded) should be displayed next to the prices or not. It can be turned to "Yes" or "No". By default, it is set to "yes".

Reviewer 3: ```suggestion
 - **Display tax label (e.g. "Tax incl.")** - This toggle button allows choosing whether the tax status (included or excluded) should be displayed next to the prices or not. It can be turned to "Yes" or "No". By default, it is set to "Yes".
```
